shell.nix

```diff
@@ -1,23 +1,32 @@
-{ nixpkgs ? import <nixpkgs> {}, compiler ? "ghc7101" }:
+{ nixpkgs ? import <nixpkgs> {}, compiler ? "default" }:
```

Comment:
this shell.nix is automatically generated by the cabal2nix tool. This new shell.nix will work with the 7.10.2 version of the compiler.